packages/ui/components/data-table/filters/MultiSelectFilterOptions.tsx

```diff
@@ -0,0 +1,76 @@
+import { classNames } from "@calcom/lib";
+import { useLocale } from "@calcom/lib/hooks/useLocale";
+import {
+  Command,
+  CommandInput,
+  CommandList,
+  CommandEmpty,
+  CommandItem,
+  CommandSeparator,
+  CommandGroup,
+  buttonClasses,
+  Icon,
+} from "@calcom/ui";
+
+import type { FilterableColumn, SelectFilterValue } from "./types";
+
+export type MultiSelectFilterOptionsProps = {
+  column: FilterableColumn;
+  filterValue?: SelectFilterValue;
+  setFilterValue: (value: SelectFilterValue) => void;
+  removeFilter: (columnId: string) => void;
+};
+
+export function MultiSelectFilterOptions({
+  column,
+  filterValue,
+  setFilterValue,
+  removeFilter,
+}: MultiSelectFilterOptionsProps) {
+  const { t } = useLocale();
+
+  return (
```

Comment:
This part is extracted from `packages/ui/components/data-table/filters/index.tsx`. The markups are almost the same, but only `onSelect` has changed a bit.